modules/atomic-widgets/widget-styles.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets;
+
+use Elementor\Modules\AtomicWidgets\Base\Style_Transformer_Base;
+use Elementor\Core\Files\CSS\Post;
+use Elementor\Element_Base;
+use Elementor\Modules\AtomicWidgets\Base\Atomic_Widget_Base;
+use Elementor\Modules\AtomicWidgets\Styles\Styles_Renderer;
+use Elementor\Modules\AtomicWidgets\Styles\Transformers\Array_Transformer;
+use Elementor\Modules\AtomicWidgets\Styles\Transformers\Size_Transformer;
+use Elementor\Plugin;
+
+class Widget_Styles {
+	/**
+	 * @var array<string, Style_Transformer_Base> $transformers
+	 */
+	private array $style_transformers = [];
```

Comment:
I think it makes more sense that the transformers will be stored in the `Styles_Renderer`, I am not sure why we pass the renderer a config instead of storing it there...